jetstream/maxtext/http-server/http_server.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""HTTP Server to interact with MaxText + JetStream Server."""
+
+import asyncio
+import concurrent.futures
+import json
+import logging
+from typing import Optional
+
+import fastapi
+import grpc
+from jetstream.core.proto import jetstream_pb2
+from jetstream.core.proto import jetstream_pb2_grpc
+import pydantic
+
+
+class GenerateRequest(pydantic.BaseModel):
+  server: Optional[str] = "127.0.0.1"
+  port: Optional[str] = "9000"
```

Comment:
Should we default to port 8000 here too to keep it consistent?

Reply:
to be consistent with which one? this is the default one in Maxtext and jetStream repo

Reply:
The maxengine server uses exposes 9000 so we have to communicate via 9000 as well - https://github.com/google/maxtext/blob/main/MaxText/maxengine_server.py#L46.

Reply:
If this is pointing to the GRPC port, that makes sense. I thought we had a different default for the http port here.

Reply:
yep, thanks for the suggestion